typings/objection/index.d.ts

```diff
@@ -471,6 +471,7 @@ declare namespace Objection {
     // Non-query methods:
 
     context(queryContext: object): this;
+    context(): object;
```

Comment:
Yes they are pretty useless, I just took the same type out that was passed in in existing `context(queryContext: object): this;` Is there any problems if I change both of them to `QueryContext` type?